src/hello-corectl/hello-visualization-corectl/linechart.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-env browser */
```

Comment:
Is there a way to import the files from hello-visualization so that we don't have so many copied files?